_includes/manuals/1.12/5.8.3_secret_token.md

```diff
@@ -0,0 +1,9 @@
+
+The last file required to make a Foreman cluster work is `$app_root/config/initializers/local_secret_token.rb`. This is used by `$app_root/config/initializers/secret_token.rb` to sign cookies. This should be set the same across all Foreman servers in your cluster. Once you have set `local_secret_token.rb`, restart Foreman and clear Foreman's cache:
```

Comment:
The "used by ".../secret_token.rb" is obsolete here, you could just say ", which is used to sign cookies."?

Thanks for checking it and updating, that's much better!